ci/unit/test_main_node.sh

```diff
@@ -22,9 +22,19 @@ sudo ln -sf "$SL_PREFIX/share/spdk/scripts/setup.sh" /usr/share/spdk/scripts
 sudo ln -sf "$SL_PREFIX/share/spdk/scripts/common.sh" /usr/share/spdk/scripts
 sudo ln -s "$SL_PREFIX/include"  /usr/share/spdk/include
 
-# set CMOCKA envs here
-export CMOCKA_MESSAGE_OUTPUT=xml
-export CMOCKA_XML_FILE="$DAOS_BASE"/test_results/%g.xml
+if [ "$WITH_VALGRIND" = "disabled"  ]; then
+    # set CMOCKA envs here
+    export CMOCKA_MESSAGE_OUTPUT=xml
+    export CMOCKA_XML_FILE="$DAOS_BASE"/test_results/%g.xml
+fi
+
```

Comment:
Why disable the JUnit result for UT when valgrind is enabled?

Reply:
Oh, I now understand. Okay, I'm going to keep the junit test results, this could be helpful for debugging if there's an error in the memcheck stage.